### PR TITLE
a lot of meta features

### PR DESCRIPTION
**Pull request by stamepicmorg**
- https://github.com/rehlds/.github/issues/23

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    url: https://github.com/dreamstalker/ReHLDS/wiki
+    about: "Reverse-engineered HLDS Documentation"
+  - name: "ReHLDS on dev-cs.ru (RU)"
+    url: https://dev-cs.ru/resources/64/
+    about: "Thread at community forum"
+  - name: "ReHLDS on cs.rin.ru (EN)"
+    url: https://cs.rin.ru/forum/viewtopic.php?f=29&t=67651
+    about: "Thread at community forum"
+  - name: "ReHLDS site"
+    url: https://rehlds.ru/
```
Comment (wopox1337) [inline .github/ISSUE_TEMPLATE/config.yml:28]: The resource https://rehlds.ru is in no way affiliated with the ReHLDS organization, not trusted by the developers.
```suggestion

```

File: .github/ISSUE_TEMPLATE/template-bug.yml [hunk 2]
```diff
@@ -0,0 +1,179 @@
+name: Bug Report
+description: A generic bug report
+title: "[BUG]: "
+labels: ["Type: Bug", "Status: Available"]
```
Comment (wopox1337) [inline .github/ISSUE_TEMPLATE/template-bug.yml:4]: Will it work with new labels? 🤨

File: .github/ISSUE_TEMPLATE/template-feature.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+name: New Feature
+description: Suggest an idea for this project
+title: "[Feature request]: "
+labels: ["Engine: Independent", "Priority: Medium", "OS: Independent", "Status: Available", "Type: Rrequest"]
```
Comment (wopox1337) [inline .github/ISSUE_TEMPLATE/template-feature.yml:4]: Is it compatible with the new labels?

File: .github/ISSUE_TEMPLATE/template-question.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    attributes:
+      label: Tell us more about your Question
+      description: What happened?
+      placeholder: Tell us about your Question!
+      value: I have an Question!
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: Thanks for completing our form!
```
Comment (wopox1337) [inline .github/ISSUE_TEMPLATE/template-question.yml:28]:
```suggestion
      value: Thanks for completing our form!

```

File: .github/ISSUE_TEMPLATE/template-task.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    attributes:
+      label: Tell us more about your task
+      description: Tell us what you want from the project and the developers?
+      placeholder: Tell us about your task!
+      value: I have an task!
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: Thanks for completing our form!
```
Comment (wopox1337) [inline .github/ISSUE_TEMPLATE/template-task.yml:28]:
```suggestion
      value: Thanks for completing our form!

```

**Review by wopox1337 — CHANGES_REQUESTED** [review]
I think it's better to have the report contain the answers to the console commands:
- `version`
- `game version`
- `meta list`

I have previously approved a similar template here:
https://dev-cs.ru/forums/server-errors/post-thread - and it works successfully for several years.

File: .github/ISSUE_TEMPLATE/template-bug.yml (hunk 2 above)
Comment (stamepicmorg) [reply .github/ISSUE_TEMPLATE/template-bug.yml:4]: nope, it should be fixed by me in few days)

File: .github/ISSUE_TEMPLATE/template-feature.yml (hunk 3 above)
Comment (stamepicmorg) [reply .github/ISSUE_TEMPLATE/template-feature.yml:4]: nope, it should be fixed by me in few days)

branch is outdated now and need to be updated

**Review by wopox1337 — APPROVED** [review]

**Pull request merged**